### PR TITLE
CALLCENTER-4839: Select2 element performs double AJAX queries to Leos.Address

### PR DESCRIPTION
**Pull request by Waldz**
https://jira.lamoda.ru/browse/CALLCENTER-4839

### CODE REVIEW

File: select2.js [hunk 1]
```diff
@@ -797,9 +797,7 @@ the specific language governing permissions and limitations under the Apache Lic
             }
 
             installKeyUpChangeEvent(search);
-            search.on("keyup-change input paste", this.bind(function () {
-                this.updateResults(true)
-            }));
+            search.on("keyup-change input paste", this.bind(this.updateResults));
```
Comment (xytis) [inline select2.js:800]: This is not nice, please explicitly bind the Boolean parameter.

Comment (Waldz) [inline select2.js:800]: Yes not nice for me too.
I just reverted to version that works. and dont want touch "their magic"

Comment (xytis) [inline select2.js:800]: So You are sure this works? Because I may not comprehend the impact of this change.

Comment (Waldz) [inline select2.js:800]: Yes, I it is just revert of my code back to their code

**Pull request merged**